test.ts

```diff
@@ -40,4 +40,2502 @@ await client.send({
   `,
 });
 
+
+await client.send({
+  from: MAIL_USER,
+  to: MAIL_TO_USER,
+  subject: "Deno Smtp build Success" + Math.random() * 1000,
+  content: "plain text email with attachment",
+  html: "<a href='https://github.com'>Github</a> and some attachment",
+  attachments: [
+    {
+      contentType: "application/pdf",
+      fileName: "test.pdf",
+      encoding: "base64",
+      data: `JVBERi0xLjMKJcTl8uXrp/Og0MTGCjMgMCBvYmoKPDwgL0ZpbHRlciAvRmxhdGVEZWNvZGUgL0xl
```

Comment:
Couldn't we use a smaller file to test? This one seems pretty big.

Reply:
Very good point... guess some simple text attachment can do it as well.